cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -1706,43 +1706,46 @@ impl_runtime_apis! {
 				Config as XcmBridgeHubRouterConfig,
 			};
 
+			use testnet_parachains_constants::rococo::locations::{AssetHubParaId, AssetHubLocation};
 			parameter_types! {
 				pub ExistentialDepositAsset: Option<Asset> = Some((
 					TokenLocation::get(),
 					ExistentialDeposit::get()
 				).into());
-				pub const RandomParaId: ParaId = ParaId::new(43211234);
+
+				pub RandomParaId: ParaId = ParaId::new(43211234);
 			}
 
 			use pallet_xcm::benchmarking::Pallet as PalletXcmExtrinsicsBenchmark;
 			impl pallet_xcm::benchmarking::Config for Runtime {
 				type DeliveryHelper = (
-					cumulus_primitives_utility::ToParentDeliveryHelper<
+				polkadot_runtime_common::xcm_sender::ToParachainDeliveryHelper<
 						xcm_config::XcmConfig,
 						ExistentialDepositAsset,
-						xcm_config::PriceForParentDelivery,
+						PriceForSiblingParachainDelivery,
+						RandomParaId,
+						ParachainSystem
 					>,
-					polkadot_runtime_common::xcm_sender::ToParachainDeliveryHelper<
+				polkadot_runtime_common::xcm_sender::ToParachainDeliveryHelper<
 						xcm_config::XcmConfig,
 						ExistentialDepositAsset,
 						PriceForSiblingParachainDelivery,
-						RandomParaId,
-						ParachainSystem,
-					>
-				);
+						AssetHubParaId,
+						ParachainSystem
+					>);
 
 				fn reachable_dest() -> Option<Location> {
-					Some(Parent.into())
+					Some(AssetHubLocation::get())
```

Comment:
@EleisonC nit, from the AssetHub's point of view using `AssetHubLocation` does not really make sense (pointing to itself), we need to use here `RandomParaId` something like this:
```suggestion
					Some(Location::new(1, [Parachain(RandomParaId::get())]))
```
Maybe it would be worth to create const `RandomParaLocation` and reuse bellow also.

Please do this change here and also for AssetHubWestend - other chains are ok using AssetHubLocation.